Dockerfile.i386

```diff
@@ -0,0 +1,60 @@
+FROM i386/ubuntu:18.04
+
+ARG S6_OVERLAY_VERSION=v2.2.0.3
+ARG S6_OVERLAY_ARCH=x86
+ARG PLEX_BUILD=linux-x86
+ARG PLEX_DISTRO=debian
+ARG DEBIAN_FRONTEND="noninteractive"
+ENV TERM="xterm" LANG="C.UTF-8" LC_ALL="C.UTF-8"
+
+ENTRYPOINT ["/init"]
+
+RUN \
+# Update and get dependencies
+    apt-get update && \
+    apt-get install -y \
+      tzdata \
+      curl \
+      xmlstarlet \
+      uuid-runtime \
+      unrar \
```

Comment:
I see the OpenCL deps are missing.  I presume that it's not available on i386?

Reply:
Correct. I don't think `armv7` has it either